Pipfile

```diff
@@ -11,8 +11,11 @@ black = "*"
 mypy = "*"
 pylint = "*"
 isort = "*"
+pre-commit = "*"
```

Comment:
Is it used for anything?

Reply:
That is weird, I have 0 recollection of using or adding pre-commit...  will remove.